src/Build/Globbing/MSBuildGlob.cs

```diff
@@ -144,7 +144,7 @@ private string NormalizeMatchInput(string stringToMatch)
 
             // Degenerate case when the string to match is empty.
             // Ensure trailing slash because the fixed directory part has a trailing slash.
-            if (stringToMatch == string.Empty)
+            if (string.IsNullOrEmpty(stringToMatch))
```

Comment:
Both callers of `NormalizeMatchInput` guarantee `stringToMatch` ensure it isn't null. I suppose one could make the argument that we may call this function in the future and potentially pass null in.